### PR TITLE
constraint input value to [-127, +127]

### PR DESCRIPTION
**Pull request by tpoisonooo**
若浮点输入或者权重小于 -max 值，按原本截断规则会出现 -128。
按照 `convolution_sgemm_int8.h`的写法：

>279                     "smull    v8.8h, v4.8b, v0.8b        \n"
 280                     "smull    v9.8h, v4.8b, v1.8b        \n"
 281                     "smull    v10.8h, v4.8b, v2.8b       \n"
 282                     "smull    v11.8h, v4.8b, v3.8b       \n"
 283
 284                     "prfm     pldl1keep, [%4, #128]      \n"
 285                     "prfm     pldl1keep, [%5, #128]      \n"
 286
 287                     "smlal2   v8.8h, v4.16b, v0.16b      \n"

279 行两个`-128`相乘得到 `2^14`，287行再累加得到 `2^15`。
超过了 int16 能表示的最大值 `2^15-1`。

风闻...模型去掉 -128 能提升精度。

### CODE REVIEW

File: src/layer/arm/requantize_arm.cpp [hunk 1]
```diff
@@ -370,4 +370,4 @@ int Requantize_arm::forward(const Mat& bottom_blob, Mat& top_blob, const Option&
     return 0;
 }
 
-} // namespace ncnn
```
Comment (nihui) [inline src/layer/arm/requantize_arm.cpp]: keep the newline at the end

File: src/layer/requantize.cpp [hunk 2]
```diff
@@ -192,4 +192,4 @@ int Requantize::forward(const Mat& bottom_blob, Mat& top_blob, const Option& opt
     return 0;
 }
 
-} // namespace ncnn
```
Comment (nihui) [inline src/layer/requantize.cpp]: keep the newline at the end

File: src/layer/x86/convolutiondepthwise_3x3_int8.h [hunk 3]
```diff
@@ -411,4 +411,4 @@ static void convdw3x3s2_int8_requant_sse(const Mat &bottom_blob, Mat &top_blob,
             r2 += tailstep;
         }
     }
-}
```
Comment (nihui) [inline src/layer/x86/convolutiondepthwise_3x3_int8.h]: keep the newline at the end

File: src/layer/arm/requantize_arm.cpp [hunk 4]
```diff
@@ -370,4 +370,5 @@ int Requantize_arm::forward(const Mat& bottom_blob, Mat& top_blob, const Option&
...
\ No newline at end of file
+} // namespace ncnn
+
```
Comment (nihui) [inline src/layer/arm/requantize_arm.cpp]: keep the unix line endings

File: src/layer/requantize.cpp [hunk 5]
```diff
@@ -192,4 +192,5 @@ int Requantize::forward(const Mat& bottom_blob, Mat& top_blob, const Option& opt
...
\ No newline at end of file
+} // namespace ncnn
+
```
Comment (nihui) [inline src/layer/requantize.cpp]: keep the unix line endings

File: src/layer/x86/convolutiondepthwise_3x3_int8.h [hunk 6]
```diff
@@ -411,4 +411,5 @@ static void convdw3x3s2_int8_requant_sse(const Mat &bottom_blob, Mat &top_blob,
...
\ No newline at end of file
+}
+
```
Comment (nihui) [inline src/layer/x86/convolutiondepthwise_3x3_int8.h]: keep the unix line endings

Comment (tpoisonooo) [reply src/layer/x86/convolutiondepthwise_3x3_int8.h]: OK, mac vim editor problem. It has been solved.

**Pull request merged**